server/config/mongoose.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+
```

Comment:
too much space just one line space is enough

Reply:
Removed Spaces